src/extension.ts

```diff
@@ -107,6 +112,8 @@ export async function activate(context: vscode.ExtensionContext) {
 
 	context.subscriptions.push(new ReleaseNotes(context, commandManager, logger));
 
+	await ensureDaemonStarted(logger);
```

Comment:
I wonder should we really do it here? It seems we should retry it sometimes based on events or something from ExtensionServiceServer?